pkg/cmd/roachtest/test_runner.go

```diff
@@ -1736,56 +1745,90 @@ func (r *testRunner) teardownTest(
 	return nil
 }
 
-// inspectArtifacts inspects node logs and attempts to write helpful triage
-// information to the test log and testRunner.githubMessage
+// inspectArtifacts inspects logs and attempts to write helpful triage
+// information to the test log and testRunner to be used in github issues.
 // This method is best effort and should not fail a test.
 // This method writes to both testLogger which is expected to be test.log and
 // t.L() which is test-teardown.log since inspectArtifacts is called after
 // teardownTest
 func (r *testRunner) inspectArtifacts(
-	ctx context.Context, t *testImpl, testLogger *logger.Logger,
-) error {
+	ctx context.Context, t *testImpl, c *clusterImpl, testLogger *logger.Logger,
+) (inspectArtifactsErr error) {
 
 	if t.Failed() || roachtestflags.AlwaysCollectArtifacts {
 		t.L().Printf("Attempting to gather node fatal level logs for triage.")
-		out, err := gatherFatalNodeLogs(t, testLogger)
-		if err != nil {
-			return err
-		}
-		if out == "" {
+		fatalOut, gatherFatalErr := gatherFatalNodeLogs(t, testLogger)
+		if gatherFatalErr != nil {
+			// even if we encounter an error continue on, this is best effort
+			inspectArtifactsErr = errors.Join(inspectArtifactsErr, gatherFatalErr)
+		} else if fatalOut == "" {
 			t.L().Printf("No fatal level logs found.")
-			return nil
 		} else {
 			testLogger.PrintfCtx(ctx, "CockroachDB contains Fatal level logs. Up to the first 10 "+
-				"will be shown here. See node logs in artifacts for more details.\n%s", out)
-			t.appendGithubMessage(out)
-			return nil
+				"will be shown here. See node logs in artifacts for more details.\n%s", fatalOut)
+			t.appendGithubFatalLogs(fatalOut)
+		}
+
+		t.L().Printf("Attempting to gather ip node mapping")
+		ipNodeMapOut, gatherNodeIpErr := gatherNodeIpMapping(t, c)
+		if gatherNodeIpErr != nil {
+			inspectArtifactsErr = errors.Join(inspectArtifactsErr, gatherNodeIpErr)
+		} else {
+			t.appendGithubIpToNodeMapping(ipNodeMapOut)
 		}
 	}
-	return nil
+	return inspectArtifactsErr
+}
+
+// gatherNodeIpMapping attempts to gather cluster node ip information for debug
+func gatherNodeIpMapping(t *testImpl, c *clusterImpl) (string, error) {
+	var table [][]string
+	cachedCluster, err := getCachedCluster(c.name)
+	if err != nil {
+		return "", err
+	}
+	// The regex to select out this table in issues.roachtestNodeToIpRE is not
+	// strict on the number of columns i.e. columns can be added / removed
+	// without a regex change
+	table = append(table, []string{"Node", "Private IP", "Public IP", "Machine Type"})
+	for _, vmInstance := range cachedCluster.VMs {
+		table = append(table, []string{vmInstance.Name, vmInstance.PrivateIP, vmInstance.PublicIP, vmInstance.MachineType})
```

Comment:
In hindsight, we probably don't need the machine type since its static, i.e. I can just look at the test to figure it out

Reply:
Agreed, machine type is easily inferred and not really relevant in majority of (triage) cases. Let's remove.